java/src/com/google/template/soy/soytree/RawTextNode.java

```diff
@@ -38,13 +38,15 @@
 
 
   /** The special chars we need to re-escape for toSourceString(). */
-  private static final Pattern SPECIAL_CHARS_TO_ESCAPE = Pattern.compile("[\n\r\t{}]");
+  private static final Pattern SPECIAL_CHARS_TO_ESCAPE = Pattern.compile("([\n\r\t{}]|/\\*\\*|\\*/)");
```

Comment:
Is it sufficient to escape `/*`?  `*/` by itself isn't a token, and IIRC `/*...*/` is a comment though not a doc comment.